server/polar/order/service.py

```diff
@@ -314,28 +330,20 @@ async def create_order_from_stripe(
             if user is None:
                 assert invoice.customer_email is not None
                 user = await user_service.get_by_email_or_signup(
-                    session, invoice.customer_email, signup_type=UserSignupType.backer
+                    session,
+                    invoice.customer_email,
+                    signup_attribution=UserSignupAttribution(
+                        intent="order",
+                        attribution=f"order:{order.id}",
```

Comment:
I've seen your attribution format with colons. Wondering how easy/hard it'll be to use that in SQL queries and such? If it becomes complex with several items (like for `utm`), I fear we'll have a hard time, no?